package.json

```
@@ -2,7 +2,7 @@
   "name": "pulsar",
   "author": "Pulsar-Edit <admin@pulsar-edit.dev>",
   "productName": "Pulsar",
-  "version": "1.109.0-dev",
+  "version": "1.110.0",
```

```suggestion
  "version": "1.110.0-dev",
```

Doing this as a code suggestion is probably the easiest way to get the `-dev` added back to the version. Heh.

Any way we do that, I would Approve, not that we really need Approvals on the version bump PRs since we're generally multiple people on-deck in real-time anyhow for these releases...